Adapters/Vungle/ISVungleAdapter/ISVungleBannerAdapterRouter.m

```diff
@@ -0,0 +1,212 @@
+//
+//  ISVungleBannerAdapterRouter.m
+//  ISVungleAdapter
+//
+//  Copyright © 2020 ironSource. All rights reserved.
+//
+
+#import "ISVungleBannerAdapterRouter.h"
+#import "ISVungleAdapter.h"
+
+@interface ISVungleBannerAdapterRouter ()
+
+@property (nonatomic, strong, nullable) NSString *bidPayload;
+
+@end
+
+@implementation ISVungleBannerAdapterRouter
+
+- (instancetype)initWithPlacementID:(NSString *)placementID
+                      parentAdapter:(ISVungleAdapter *)parentAdapter
+                           delegate:(id<ISBannerAdapterDelegate>)delegate {
+    if (self = [super init]) {
+        _placementID = placementID;
+        _parentAdapter = parentAdapter;
+        _delegate = delegate;
+        _bannerState = UNKNOWN;
+        _bidPayload = nil;
+        self.bannerSize = BannerSizeRegular;
+    }
+
+    return self;
+}
+
+- (void)loadBannerAd {
+    self.bannerAd = [[VungleBanner alloc] initWithPlacementId:self.placementID size:self.bannerSize];
+    self.bannerAd.delegate = self;
+    self.bannerAd.enableRefresh = NO;
+    self.adView = [[UIView alloc] initWithFrame:[self getAdViewRect:self.bannerSize]];
+
+    if ([self.bannerAd canPlayAd]) {
+        LogInternal_Internal(@"Banner ad: %@ is loaded", self.placementID);
+        [self showBannerAd];
+        [self.delegate adapterBannerDidLoad:self.adView];
+        return;
+    }
```

Comment:
I think L40-L45 isn't needed because the ad would never be ready until a load call is made because there's no auto-caching.

Reply:
@stanley-vungle  Thanks a lot for the guidance, updated and add some comments. :)